src/App.js

```diff
@@ -854,14 +854,18 @@ export class App extends React.Component<AppProps, AppState> {
     handleRootFocus = (e: SyntheticKeyboardEvent<HTMLElement>) => {
         // $FlowFixMe getAttribute is missing in event target
         const ariaLabel = e.target.getAttribute('aria-label');
-        if (ariaLabel) {
-            this.audioManager.playPreviewAnnouncement(ariaLabel);
-        } else {
-            // $FlowFixMe innerText is missing in event target
-            const innerText = e.target.innerText;
-            if (innerText) {
-                this.audioManager.playPreviewAnnouncement(innerText);
+        if (!this.audioManager.getFeedbackIsPlaying()) {
+            if (ariaLabel) {
+                this.audioManager.playPreviewAnnouncement(ariaLabel);
+            } else {
+                // $FlowFixMe innerText is missing in event target
+                const innerText = e.target.innerText;
+                if (innerText) {
+                    this.audioManager.playPreviewAnnouncement(innerText);
+                }
             }
+        } else {
+            this.audioManager.setFeedbackIsPlaying(false);
```

Comment:
I still think the audio manager could handle both parts of this, i.e. you should always be able to ask it to announce something and let it figure out what to do if it's already playing.